test/e2e/windows/reboot_node.go

```diff
@@ -196,6 +197,11 @@ var _ = sigDescribe(feature.Windows, "[Excluded:WindowsDocker] [MinimumKubeletVe
 			}
 		}
 
+		ginkgo.By("Checking whether node is rebooted")
```

Comment:
I think the intent of https://github.com/kubernetes/kubernetes/issues/108886 was to remove the fragile loop above that uses a pod restart to determine if the node was restarted with an `gomega.Eventually` loop that would check for the boot id getting incremented.  This would be more reliable than check to see if a pod restarted.

We can keep this check as well (though it will be implicit in the `gomega.eventually` check)